plugins/woocommerce-blocks/assets/js/blocks/product-collection/constants-register-product-collection.ts

```diff
@@ -0,0 +1,154 @@
+/**
```

Comment:
I suggest renaming this file to `constants.ts` as it actually contains constants while the other file (currently `constants.ts`) contains more of a `utils`.